full-service/src/bin/main.rs

```diff
@@ -47,14 +47,14 @@ fn main() {
                 config.wallet_db, err
             )
         });
-
     embedded_migrations::run(&conn).expect("failed running migrations");
 
     let wallet_db = WalletDb::new_from_url(
         config
             .wallet_db
             .to_str()
             .expect("Could not get wallet_db path"),
+        10,
```

Comment:
I mean, 10 sounds good to me (though I doubt we'll ever hit more than 2), is this just a feel good number or is it a standard you came across for pooled connections?

Reply:
It's the default for pooled connections as shipped by diesel - but I don't have any other reason to set 10, perhaps when we're doing performance testing we'll figure out what's most optimal here